.eleventy.js

```diff
@@ -1,7 +1,6 @@
-const { isDev } = require('./pages/views/_data/env');
-const markdownIt = require("markdown-it")({
-  html: true
-});
+const markdown = require('./pages/11y/markdown');
```

Comment:
```suggestion
const markdown = require('./pages/11ty/markdown');
```
And move to appropriate folder